packages/gestalt/src/style.js

```diff
@@ -75,6 +75,5 @@ export const toProps = ({ className, inlineStyle }: Style): ToPropsOutput => {
     props.style = inlineStyle;
   }
 
-  // $FlowFixMe[incompatible-exact]
-  return props;
+  return { ...props };
```

Comment:
This is the only possible runtime change. It _shouldn't_ matter, but there is the possibility